### PR TITLE
fixing cyberduck process

### PR DESCRIPTION
**Pull request by anthonyvetter**
card: https://trello.com/c/TGMTSmRI/1605-fix-steps-in-kb-using-object-storage-
from-3rd-party-tools-for-aws2-authentication

### CODE REVIEW

File: Object Storage/using-object-storage-from-3rd-party-tools.md [hunk 1]
```diff
@@ -54,25 +54,31 @@ CenturyLink Cloud now has a robust, geo-distributed Object Storage service capab
...
    ![Access Keys](../images/kb-object-storage-access-keys.png)
 
-4. Also you will need the **Service Point** (aka server) where your bucket exists (or will exist). In the Control Portal, on the Object Storage bucket listing page, either create a new bucket, or click on an existing bucket. When the **Bucket Info and Settings** page is displayed, copy/save the portion of the **bucket URL** that follows the bucket name. In this example, the bucket **Service Point** (server name) is: **canada.os.ctl.io**.
+5. Also you will need the **Service Point** (aka server) where your bucket exists (or will exist). In the Control Portal, on the Object Storage bucket listing page, either create a new bucket, or click on an existing bucket. When the **Bucket Info and Settings** page is displayed, copy/save the portion of the **bucket URL** that follows the bucket name. In this example, the bucket **Service Point** (server name) is: **canada.os.ctl.io**.
    ![object storage bucket info](../images/object-storage-bucket-info-and-settings.png)
```
Comment (chrislittle) [inline Object Storage/using-object-storage-from-3rd-party-tools.md]: check the preview as this isn't wrapping properly.  best to add a full line between the text and the image

File: Object Storage/using-object-storage-from-3rd-party-tools.md [hunk 2]
```diff
@@ -54,25 +54,31 @@ CenturyLink Cloud now has a robust, geo-distributed Object Storage service capab
...
+
+  1. For creating connections later, replace steps the previous 2 steps with this: Click the **Open Connection** icon (upper left corner) in order to configure the connection details for your Object Storage bucket. Select **S3 (Amazon Simple Storage)** from the drop down. In the **Server** field, enter the name of the **Service point**. Leave Port field set to **443**. Populate the **Username** field with your **Access Key ID** and populate the **Password** field with your **Secret Access key**.
    ![cyberduck setup connection](../images/cyberduck-setup-connection.png)
 
-6. After you have configured the Object Storage connection, CyberDuck will list the available buckets at that **Connection**. You can double-click on a bucket to upload files or even drag-and-drop files into the bucket.
+8. After you have configured the Object Storage connection, CyberDuck will list the available buckets at that **Connection**. You can double-click on a bucket to upload files or even drag-and-drop files into the bucket.
    ![cyberduck established connection](../images/cyberduck-established-connection.png)
```
Comment (chrislittle) [inline Object Storage/using-object-storage-from-3rd-party-tools.md]: check the preview as this isn't wrapping properly.  best to add a full line between the text and the image

**Comment by anthonyvetter** [comment]
Good catch.  I did't actually add any images, or remove any lines, I did move some blocks of text around though.   So I didn't check that.  After looking at the preview I do see the issue and added the extra lines.  Submitting another pull request shortly.

**Comment by anthonyvetter** [comment]
err... fixed and sync'd

**Comment by anthonyvetter** [comment]
In my markup preview in Atom, everything looks fine.  But something with how the website translates markup is causing a nested list to be bolded.  Any ideas what is causing this?  

https://www.ctl.io/knowledge-base/object-storage/using-object-storage-from-3rd-party-tools/
Bolded block starting at: "For creating connections later, replace steps the previous 2 steps with this: Click the Open Connection icon..."

code:
'''
7. Open the connection by clicking on the bookmark.  Populate the **Password** field with your **Secret Access key**.  Then click **Login**.

  1. For creating connections later [...]
'''

**Pull request merged**